engine/src/main/java/org/destinationsol/common/SolMath.java

```diff
@@ -431,7 +433,8 @@ public static void checkVectorsTaken(Object o) {
     }
 
     /**
-     * @return solution of a quadratic equation. if 2 solutions possible, the greater is returned.
+     * Solves a quadratic equation (equation in form {@code 0 = (a * x^2) + (b * x) + c}
+     * @return Solution to a quadratic equation. When two solutions are possible, the greater of them is returned.
      */
     public static float genQuad(float a, float b, float c) {
```

Comment:
Can you rename this function as well? Make it something more explicit/meaningful.

Reply:
@vampcat What about solveQuadraticEquation()